qualtran/qref_interop/_bloq_to_qref_test.py

```diff
@@ -0,0 +1,261 @@
+from dev_tools.qualtran_dev_tools.bloq_finder import get_bloq_examples
```

Comment:
@mpharrigan This is not part of the qualtran package; so we should probably not use it in tests to avoid adding a dependency.

Reply:
Yeah, @mstechly called it out in the original PR description. For the tests, I'd recommend picking a few key examples to test the qref interop functionality and then later we can add 

 - a qref conversion check to the "bloq report card" which is run ad-hoc
 - a qref conversion check to the `bloq_autotester` pytest fixture, which will run against most bloq examples

Reply:
Ok, I've chosen some examples (open to using other examples) and changed the code.